### ISSUE TITLE
Minimap colours not updating on stylesheet modification

### ISSUE THREAD

**Issue by Siguza** [atom-minimap/minimap#303]
The title says it all: If you change your (user-)stylesheet, the editor itself reloads all colours, the minimap doesn't.

Tested with

    ::shadow
    {
        .source, .java.meta.class
        {
            color: #FFF;
        }
    }

**Comment by fundon** [comment]
Thank you for your feedback.
We will check this.

**Comment by abe33** [comment]
Nice catch, not sure if there's an event on user stylesheet reload, but if there is, we just have to hook on it.

**Comment by fundon** [comment]
https://atom.io/docs/api/v0.188.0/StyleManager
There is a `StyleManager` API and Atom has `atom.styles` object.

**Comment by fundon** [comment]
@abe33 
Use of `@subscriptions.add atom.styles.onDidAddStyleElement =>` instead of `@subscriptions.add atom.themes.onDidChangeActiveThemes` and it's working.

```coffee
@subscriptions.add atom.styles.onDidAddStyleElement =>
      @invalidateCache()
      @requestForcedUpdate()
```

**Comment by fundon** [comment]
Landed in v4.7.0
